README.md

```diff
@@ -8,32 +8,39 @@ This is an [embedded-hal](https://github.com/rust-embedded/embedded-hal) driver
 
 ```rust
 // ...
-
 use bmi2::Bmi2;
 use bmi2::config;
 use bmi2::{types::Burst, I2cAddr, types::PwrCtrl};
+use embedded_hal::delay::DelayNs;
+
+// Specify your delay type, for example:
+let delay = MyDelay::new(); // Replace with your actual delay implementation
+
+// Choose a buffer size (e.g., 512 bytes), needs to be >= max data burst
+const BUFFER_SIZE: usize = 512;
 
 /// Create a new Bmi2 device using I2C with its alternative address (0x69).
 /// Configure the max data burst to 255 bytes:
 /// - used for the upload of the configuration during initialization.
-/// - This is a limitation from your device or its HAL. 
-let mut bmi = Bmi2::new_i2c(i2c, I2cAddr::Alternative, Burst::Other(255));
+/// - This is a limitation from your device or its HAL.
+let mut bmi = Bmi2::<_, _, BUFFER_SIZE>::new_i2c(
+    i2c, 
+    I2cAddr::Alternative, 
+    Burst::Other(255),
+    delay
```

Comment:
Here `delay` should be the 2nd argument not the last one and it should be `Burst::new` instead of `Burst::Other`. Otherwise LGTM!

Reply:
Good catch, will fix it!

Reply:
I've updated the Readme and the basic_read example in the latest commit.